litellm/integrations/opentelemetry.py

```diff
@@ -93,11 +91,16 @@ def __init__(
         self.OTEL_EXPORTER = self.config.exporter
         self.OTEL_ENDPOINT = self.config.endpoint
         self.OTEL_HEADERS = self.config.headers
-        provider = TracerProvider(resource=Resource(attributes=LITELLM_RESOURCE))
-        provider.add_span_processor(self._get_span_processor())
         self.callback_name = callback_name
 
-        trace.set_tracer_provider(provider)
+        if self.OTEL_EXPORTER != "none":
```

Comment:
I opted to use the value `none` to indicate that no exporter should be created. This is [similar to how opentelemetry indicates not to create a metrics exporter](https://opentelemetry.io/docs/zero-code/python/configuration/#configuration-properties).